examples/chain_client_examples/10_ExchangeMsgBatchCancelDerivativeOrders.py

```diff
@@ -34,7 +34,7 @@ async def main() -> None:
     # load account
     priv_key = PrivateKey.from_hex("5d386fbdbf11f1141010f81a46b40f94887367562bd33b452bbaa6ce1cd1381e")
     pub_key = priv_key.to_public_key()
-    address = await pub_key.to_address().init_num_seq(network.lcd_endpoint)
```

Comment:
The init_num_seq function we had before was async, not sure when we changed that in the Address class but certainly worth pointing it in our API channel. This is a breaking change for those that will use that function as async.

I'll prepare the change in the docs to push for tomorrow, let me know when the new SDK is out @DrunkRandomWalker